docs/pages/integrating/api.rst

```diff
@@ -14,16 +14,16 @@ API Reference
             const ui = SwaggerUIBundle({
                 "dom_id": "#swagger-ui",
                 urls: [
-                    {url: "../../_static/auth/iam.yaml", name: "Auth (iam)"},
-                    {url: "../../_static/auth/v1.yaml", name: "Auth (v1) - DEPRECATED"},
+                    {url: "../../_static/auth/iam.yaml", name: "Auth (v2)"},
                     {url: "../../_static/crypto/v1.yaml", name: "Crypto"},
```

Comment:
Maybe we can move Crypto and Monitoring to the bottom of the non-deprecated list? Since there are little to no users of those APIs

Reply:
Alphabetic makes more sense to me. If we want to make it easier for users to find the APIs they need we're better off using more descriptive names in stead of the module names.